decidim-core/db/migrate/20180508111710_add_accepted_tos_version_field_to_users.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class AddAcceptedTosVersionFieldToUsers < ActiveRecord::Migration[5.1]
+  class Organization < ApplicationRecord
+    self.table_name = :decidim_organizations
+    has_many :users, foreign_key: "decidim_organization_id", class_name: "Decidim::User", dependent: :destroy
+  end
+  class User < ApplicationRecord
+    self.table_name = :decidim_users
+    belongs_to :organization, foreign_key: "decidim_organization_id", class_name: "Decidim::Organization"
+  end
+
+  def up
+    add_column :decidim_users, :accepted_tos_version, :datetime
+    Organization.find_each do |organization|
+      # rubocop:disable Rails/SkipsModelValidations
+      organization.users.update_all(accepted_tos_version: organization.tos_version)
```

Comment:
Shouldn't we set a `tos_version` value to the organization first?

Reply:
Yes, there's a migration at `master` branch from PR #3491 that sets the `tos_version` to the organization:
[decidim/20180508111640_add_tos_version_to_organization.rb at master · decidim/decidim](https://github.com/decidim/decidim/blob/master/decidim-core/db/migrate/20180508111640_add_tos_version_to_organization.rb)